doc: about-this-documentation.md (#13)
mobxjs/zh.mobx.js.org: docs/about-this-documentation.md

---
title: 关于本文档
sidebar_label: 关于本文档
hide_title: true
---

<script async type="text/javascript" src="//cdn.carbonads.com/carbon.js?serve=CEBD4KQ7&placement=mobxjsorg" id="_carbonads_js"></script>

# 关于本文档

> 它遵循的原则是先介绍最常用的概念，然后再介绍专门的信息。这适用于概念表中的标题以及这些标题下的页面。

Reviewer: ```suggestion
它遵循的原则是先介绍最常用的概念，然后再介绍专门的信息。这适用于概念目录中的标题以及那些标题下的页面。
```

> 我们用 {🚀} 标记标记了更高级的部分和概念。除非您有特殊的用例，否则您可能不必了解它们，并且可以在不了解它们的情况下非常有效地使用 MobX。随时跳过它们，继续进行下一部分！

Reviewer: ```suggestion
我们用 {🚀} 记号标记了更高级的部分和概念。除非您有特殊的用例，否则您可能不必了解它们，并且您可以在不了解它们的情况下非常高效地使用 MobX。请随意跳过它们，继续阅读接下来的部分！
```

> MobX 6 的文档已重写。对于旧版本的 MobX，可以在[此处](https://github.com/mobxjs/mobx/tree/mobx4and5/docs)找到。所有原理都是相同的，API 也基本相同。主要区别在于，在 MobX 6 之前，[修饰符](https://github.com/mobxjs/mobx/blob/mobx4and5/docs/best/decorators.md) 是编写 MobX 增强类的推荐语法。

Reviewer: ```suggestion
我们为了 MobX 6 对文档进行了重写。对于旧版本的 MobX，文档可以在[此处](https://github.com/mobxjs/mobx/tree/mobx4and5/docs)找到。所有原理都是相同的，API 也基本相同。主要区别在于，在 MobX 6 之前，[修饰符](https://github.com/mobxjs/mobx/blob/mobx4and5/docs/best/decorators.md) 是编写 MobX 强化型类的推荐语法。
```

> 文档摘要可以作为备忘单下载：

Reviewer: ```suggestion
你可以把文档摘要下载为速查表：
```

> <div class="cheat"><a href="https://gum.co/fSocU"><button title="下载 MobX 6 备忘单和赞助这个项目">下载 MobX 6 备忘单</button></a></div>

Reviewer: ```suggestion
<div class="cheat"><a href="https://gum.co/fSocU"><button title="下载 MobX 6 速查表赞助这个项目">下载 MobX 6 速查表</button></a></div>
```

> ## 指引

Reviewer: ```suggestion
## 导览
```

> 要全面了解如何将 MobX 与 React 一起使用，请通读当前的 _Introduction_ 标题，特别是 [MobX 部分的要点](the-gist-of-mobx.md)。它将向您介绍最重要的原则、API以及它们之间的关系。阅读本文后，您应该已经准备好使用 MobX 了！

Author: @iChenLei  _Introduction_ 是因为现在的左侧导航栏的名称，后续可以一并修改，这里就不做改变了

> 要全面了解如何将 MobX 与 React 一起使用，请通读当前的 _Introduction_ 标题，特别是 [MobX 部分的要点](the-gist-of-mobx.md)。它将向您介绍最重要的原则、API以及它们之间的关系。阅读本文后，您应该已经准备好使用 MobX 了！

Reviewer: ```suggestion
要全面了解如何将 MobX 与 React 一起使用，请通读当前的 _Introduction_ 标题，特别是 [MobX 的要点](the-gist-of-mobx.md)。它将向您介绍最重要的原则、API以及它们之间的关系。看完这个，你应该就可以使用 MobX 了！
```

> 以下是有关接下来要检查的一些建议：

Reviewer: ```suggestion
接着，你可以看看以下推荐的内容：
```

-   尝试 [10 分钟的 Mobx 和 React 交互式介绍](https://mobx.js.org/getting-started)

-   [React 集成](react-integration.md)

-   [`makeObservable` / `makeAutoObservable`](observable-state.md)

-   了解 [actions](actions.md)，其中包括有关异步操作的讨论

-   [computeds](computeds.md) 基本知识

> -   阅读 [`autorun`](reactions.md#autorun)， 如果只是因为在示例中使用了它

Reviewer: ```suggestion
-   阅读有关 [`autorun`](reactions.md#autorun) 的内容，即使只是因为示例中用到了它
```
